tests/test-files/bad/MissingImportParens.elm

```diff
@@ -0,0 +1,4 @@
+module Main exposing (..)
+
+import ImportModuleWithoutParensWithLineBreak exposing
+   A, B
```

Comment:
Elm files are invalid if they contain no definitions...  Does this test case still fail if you add `x = ()` (on its own line) at the end of the file?

Reply:
You're right. This was the case and not the actual parsing to fail. It looks to my like parsing line breaks is working. Using the existing implementation I'm not sure what to expect regarding line breaks. Do you mind having a look into it? Is the current state of this PR working as we expect it?